### PR TITLE
Update the demos in the Examples folder.

### PR DESCRIPTION
**Pull request by LesiaChaban**

### CODE REVIEW

**Comment by LesiaChaban** [comment]
Started work on demos, as progress and editing of demos will be added for each commit.

File: docs/examples/Examples.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ However:
 * [Fibonacci](/examples/fibonacci/)
   * [Standalone Fibonacci](/examples/standalone_fibonacci/)
 * [Styled Text](/examples/styled_text/)
-* [Astronauts](/examples/astronauts/)
+* [Requests Module](/examples/requests_module/)
```
Comment (bhoran) [inline docs/examples/Examples.md:13]:
```suggestion
* [Using the Requests Module](/examples/requests_module/)
```

File: docs/examples/Requests_Module.md [hunk 2]
```diff
@@ -1,20 +1,27 @@
 ---
 layout: page
-title: Astronauts
-permalink: /examples/astronauts/
+title: Using Requests Module in GraalPy
```
Comment (bhoran) [inline docs/examples/Requests_Module.md:3]:
```suggestion
title: Using the Requests Module in GraalPy
```

File: docs/examples/Requests_Module.md [hunk 3]
```diff
@@ -1,20 +1,27 @@
...
+permalink: /examples/requests_module/
 ---
 
-This example provides a demonstration of using the Python [`requests`](https://pypi.org/project/requests/) module to call a REST API.
+# Using Requests Module in GraalPy
```
Comment (bhoran) [inline docs/examples/Requests_Module.md:7]: Delete this line, it duplicates the title.

File: docs/examples/Requests_Module.md [hunk 4]
```diff
@@ -1,20 +1,27 @@
...
 
-1. Install `graalpy` and [create a Virtual Environment](/guides/creating_a_virtual_environment/), then activate it. 
-(For more information, see [Getting Started](/getting_started/).)
+This example provides a demonstration of using the Python [`requests`](https://pypi.org/project/requests/) module to call a REST API on the GraalPy runtime. 
+The `requests` module is a library for making HTTP requests. 
+
+1. [Download GraalPy version 23.1.0 or higher](https://github.com/oracle/graalpython/releases/) and [install it](../getting_started.md).
+
+2. [Create a Virtual Environment](../guides/Creating_a_Virtual_Environment.md) and activate it. 
+    For more information, see [Getting Started](../getting_started.md).
```
Comment (bhoran) [inline docs/examples/Requests_Module.md:15]: Restore this, use the permalinks, not the filenames.

File: docs/examples/Requests_Module.md [hunk 5]
```diff
@@ -1,20 +1,27 @@
...
 
 2. Install the `requests` package using `pip`:
+    
+    >Note: To install `pip` in your GraalPy environment, use the `ensurepip` or `get-pip.py` [installers](https://pip.pypa.io/en/stable/installation/).
```
Comment (bhoran) [inline docs/examples/Requests_Module.md:19]: Delete this.

File: docs/examples/Requests_Module.md [hunk 6]
```diff
@@ -1,20 +1,27 @@
...
 
-This example provides a demonstration of using the Python [`requests`](https://pypi.org/project/requests/) module to call a REST API.
+# Using Requests Module in GraalPy
 
-1. Install `graalpy` and [create a Virtual Environment](/guides/creating_a_virtual_environment/), then activate it. 
-(For more information, see [Getting Started](/getting_started/).)
+This example provides a demonstration of using the Python [`requests`](https://pypi.org/project/requests/) module to call a REST API on the GraalPy runtime. 
+The `requests` module is a library for making HTTP requests. 
+
+1. [Download GraalPy version 23.1.0 or higher](https://github.com/oracle/graalpython/releases/) and [install it](../getting_started.md).
```
[the remark this answers is not in the document]
Comment (LesiaChaban) [reply docs/examples/Requests_Module.md:12]: Okay, will do!

File: docs/examples/Requests_Module.md [hunk 7]
```diff
@@ -1,20 +1,27 @@
...
+2. [Create a Virtual Environment](../guides/Creating_a_Virtual_Environment.md) and activate it. 
+    For more information, see [Getting Started](../getting_started.md).
 
 2. Install the `requests` package using `pip`:
+    
+    >Note: To install `pip` in your GraalPy environment, use the `ensurepip` or `get-pip.py` [installers](https://pip.pypa.io/en/stable/installation/).
 
     ```bash
-    % pip install requests
+    graalpy -m pip install requests
```
Comment (bhoran) [inline docs/examples/Requests_Module.md:22]: Restore this. The standard installation procedure is to use `pip`, not to call the module.

File: docs/examples/Requests_Module.md [hunk 8]
```diff
@@ -38,7 +45,7 @@ This example provides a demonstration of using the Python [`requests`](https://p
 4. Run the script:
 
     ```bash
-    % graalpy astronauts.py
+    graalpy astronauts.py
```
Comment (bhoran) [inline docs/examples/Requests_Module.md:48]: Restore this. We use the "%" to indicate a bash command throughout the guide.

File: docs/examples/Requests_Module.md [hunk 9]
```diff
@@ -1,20 +1,27 @@
...
+title: Using Requests Module in GraalPy
+permalink: /examples/requests_module/
 ---
 
-This example provides a demonstration of using the Python [`requests`](https://pypi.org/project/requests/) module to call a REST API.
+# Using Requests Module in GraalPy
 
-1. Install `graalpy` and [create a Virtual Environment](/guides/creating_a_virtual_environment/), then activate it. 
-(For more information, see [Getting Started](/getting_started/).)
+This example provides a demonstration of using the Python [`requests`](https://pypi.org/project/requests/) module to call a REST API on the GraalPy runtime.
```
Comment (bhoran) [inline docs/examples/Requests_Module.md:9]: Restore this line.

File: docs/examples/Requests_Module.md [hunk 10]
```diff
@@ -1,20 +1,27 @@
...
+permalink: /examples/requests_module/
 ---
 
-This example provides a demonstration of using the Python [`requests`](https://pypi.org/project/requests/) module to call a REST API.
+# Using Requests Module in GraalPy
 
-1. Install `graalpy` and [create a Virtual Environment](/guides/creating_a_virtual_environment/), then activate it. 
-(For more information, see [Getting Started](/getting_started/).)
+This example provides a demonstration of using the Python [`requests`](https://pypi.org/project/requests/) module to call a REST API on the GraalPy runtime. 
+The `requests` module is a library for making HTTP requests.
```
Comment (bhoran) [inline docs/examples/Requests_Module.md:10]: Delete this line. Any knowledgeable Python developer already knows what "requests" is.

File: docs/examples/Examples.md
Comment (bhoran) [inline docs/examples/Examples.md:1]: I've refactored this file.

File: docs/examples/Requests_Module.md
Comment (bhoran) [inline docs/examples/Requests_Module.md:1]: I've refactored the examples so that the filename change is no longer required.

Comment (LesiaChaban) [reply docs/examples/Requests_Module.md:1]: I will then close that PR and continue when you merge your PR into `gh-pages` branch.

Comment (LesiaChaban) [reply docs/examples/Requests_Module.md:1]: Now I will only test the demos without making any changes.

**Pull request closed without merging**